tfe/resource_tfe_variable.go

```diff
@@ -224,6 +231,18 @@ func (r *resourceTFEVariable) Schema(ctx context.Context, req resource.SchemaReq
 					),
 				},
 			},
+			"readable_value": schema.StringAttribute{
+				Optional: true,
```

Comment:
```suggestion
```

(...etc. etc., also removing the Validators and the whole custom validator implementation below)

Reply:
Wow, that is exactly what I was looking for! Thanks Nick!